kitsune/search/v2/documents.py

```diff
@@ -15,30 +19,41 @@
 
 
 class WikiDocument(SumoDocument):
-    url = field.Keyword()
-    updated = field.Date()
+    """ES Document for Knowledge Base.
+
+    Each document in ES is built on top of a parent KB article.
+    All the children are included in the same document.
+    """
 
+    updated = field.Date()
```

Comment:
Is it all that useful having the updated date of the parent, without those of the children?